From 44f7eb015bc518af80ac5a7f45a8a5c4c91a657d Mon Sep 17 00:00:00 2001
From: Sam Hartman <hartmans@debian.org>
Date: Mon, 26 Dec 2011 18:03:19 -0500
Subject: Debian manpage patch

Adjust paths in manpages to be more consistent with FHS

Remove edirectory support which is not built on Debian from LDAP man pages.

Patch-Category: debian-local
---
 src/clients/kinit/kinit.M                       |    2 +-
 src/clients/ksu/ksu.M                           |   30 ++--
 src/config-files/kdc.conf.M                     |    6 +-
 src/config-files/krb5.conf.M                    |    4 +-
 src/kadmin/cli/kadmin.M                         |   10 +-
 src/kadmin/server/kadmind.M                     |    2 +-
 src/krb5-config.M                               |   23 ++--
 src/plugins/kdb/ldap/ldap_util/kdb5_ldap_util.M |  201 -----------------------
 src/slave/kprop.M                               |    4 +-
 src/slave/kpropd.M                              |   10 +-
 src/slave/kproplog.M                            |    2 +-
 11 files changed, 47 insertions(+), 247 deletions(-)

diff --git a/src/clients/kinit/kinit.M b/src/clients/kinit/kinit.M
index 0a919c0..126794a 100644
--- a/src/clients/kinit/kinit.M
+++ b/src/clients/kinit/kinit.M
@@ -70,7 +70,7 @@ must be followed immediately by one of the following delimiters:
 .in -.3i
 .fi
 .sp
-as in "kinit -l 90m".  You cannot mix units; a value of `3h30m' will
+as in "kinit \-l 90m".  You cannot mix units; a value of `3h30m' will
 result in an error.
 .sp
 If the
diff --git a/src/clients/ksu/ksu.M b/src/clients/ksu/ksu.M
index 00e0008..905a342 100644
--- a/src/clients/ksu/ksu.M
+++ b/src/clients/ksu/ksu.M
@@ -175,28 +175,28 @@ of just exec, as done by su.
 .PP
 Ksu can be used to create a new security context for the
 target program (either the target
-shell, or command specified via the -e option).
+shell, or command specified via the \-e option).
 The target program inherits a set
 of credentials from the source user.
 By default, this set includes all of the credentials
 in the source cache plus any
 additional credentials obtained during authentication.
 The source user is able to limit the credentials in this set
-by using -z or -Z option.
--z restricts the copy of tickets from the source cache
+by using \-z or \-Z option.
+\-z restricts the copy of tickets from the source cache
 to the target cache to only the tickets where client ==
-the target principal name.  The -Z option
+the target principal name.  The \-Z option
 provides the target user with a fresh target cache
 (no creds in the cache). Note that for security reasons,
 when the source user is root and target user is non-root,
--z option is the default mode of operation. 
+\-z option is the default mode of operation. 
 
 While no authentication takes place if the source user
 is root or is the same as the target user, additional
 tickets can still be obtained for the target cache.
-If -n is specified and no credentials can be copied to the target
+If \-n is specified and no credentials can be copied to the target
 cache,  the  source user is prompted for a Kerberos password
-(unless -Z specified or GET_TGT_VIA_PASSWD is undefined). If
+(unless \-Z specified or GET_TGT_VIA_PASSWD is undefined). If
 successful,  a  TGT is obtained from the Kerberos server and
 stored in the target cache.  Otherwise,
 if a password is not provided (user hit return)
@@ -301,7 +301,7 @@ ksu deletes the target cache.
 \fB\-D
 turn on debug mode.
 .TP 10
-\fITicket granting ticket options: -l lifetime -r time -pf\fP
+\fITicket granting ticket options: \-l lifetime \-r time \-pf\fP
 The ticket granting ticket options only apply to the
 case where there are no appropriate tickets in
 the cache to authenticate the source user. In this case
@@ -341,7 +341,7 @@ if you want the tickets for other then the default
 principal. Note that the
 .B \-z 
 option is mutually
-exclusive with the -Z option.
+exclusive with the \-Z option.
 .TP 10
 \fB\-Z
 Don't copy any tickets from the source cache to the
@@ -350,7 +350,7 @@ where the default principal name of the cache is
 initialized to the target principal name.  Note that
 .B \-Z
 option is mutually
-exclusive with the -z option.
+exclusive with the \-z option.
 .TP 10
 \fB\-q
 suppress the printing of status messages.
@@ -365,7 +365,7 @@ specified command (Example of usage: ksu bob
 ls
 .B \-lag).
 
-\fIThe authorization algorithm for -e is as follows:\fP
+\fIThe authorization algorithm for \-e is as follows:\fP
 
 If the source user is root or source user == target user,
 no authorization takes place and             
@@ -418,7 +418,7 @@ path or just the program name.
 .TP 10
 \fB\-a \fIargs
 specify arguments to be passed to the target shell.
-Note: that all flags and parameters following -a
+Note: that all flags and parameters following \-a
 will be passed to the shell, thus all options
 intended for ksu must precede
 .B \-a.
@@ -449,7 +449,7 @@ have a secure channel, the password may get exposed.
 during the resolution of the default principal name,
 PRINC_LOOK_AHEAD enables ksu to find principal names          
 in the .k5users file as described in the OPTIONS section
-(see -n option).      
+(see \-n option).      
 .TP 10
 \fICMD_PATH\fP
 specifies a list of directories containing programs
@@ -463,8 +463,8 @@ the names of "legal shells".  Note that the target user's
 shell is obtained from the passwd file.
 .TP 10
 SAMPLE CONFIGURATION:
-KSU_OPTS = -DGET_TGT_VIA_PASSWD 
--DPRINC_LOOK_AHEAD -DCMD_PATH='"/bin /usr/ucb /local/bin"
+KSU_OPTS = \-DGET_TGT_VIA_PASSWD 
+\-DPRINC_LOOK_AHEAD \-DCMD_PATH='"/bin /usr/ucb /local/bin"
 .TP 10
 PERMISSIONS FOR KSU
 ksu should be owned by root and have the set user id  bit turned on.   
diff --git a/src/config-files/kdc.conf.M b/src/config-files/kdc.conf.M
index b47c251..8c83d27 100644
--- a/src/config-files/kdc.conf.M
+++ b/src/config-files/kdc.conf.M
@@ -92,14 +92,14 @@ This
 .B string
 specifies the location of the access control list (acl) file that
 kadmin uses to determine which principals are allowed which permissions
-on the database. The default value is /usr/local/var/krb5kdc/kadm5.acl.
+on the database. The default value is /etc/krb5kdc/kadm5.acl.
 
 .IP admin_keytab
 This
 .B string
 Specifies the location of the keytab file that kadmin uses to
 authenticate to the database.  The default value is
-/usr/local/var/krb5kdc/kadm5.keytab.
+/etc/krb5kdc/kadm5.keytab.
 
 .IP database_name
 This
@@ -274,7 +274,7 @@ tickets should be checked against the transit path computed from the
 realm names and the [capaths] section of its krb5.conf file
 
 .SH FILES 
-/usr/local/var/krb5kdc/kdc.conf
+/etc/krb5kdc/kdc.conf
 
 .SH SEE ALSO
 krb5.conf(5), krb5kdc(8)
diff --git a/src/config-files/krb5.conf.M b/src/config-files/krb5.conf.M
index af4200c..13b00cb 100644
--- a/src/config-files/krb5.conf.M
+++ b/src/config-files/krb5.conf.M
@@ -522,7 +522,7 @@ is specified, the default is AUTH.
 In the following example, the logging messages from the KDC will go to
 the console and to the system log under the facility LOG_DAEMON with
 default severity of LOG_INFO; and the logging messages from the
-administrative server will be appended to the file /var/adm/kadmin.log
+administrative server will be appended to the file /var/log/kadmin.log
 and sent to the device /dev/tty04.
 .sp
 .nf
@@ -530,7 +530,7 @@ and sent to the device /dev/tty04.
 [logging]
 	kdc = CONSOLE
 	kdc = SYSLOG:INFO:DAEMON
-	admin_server = FILE:/var/adm/kadmin.log
+	admin_server = FILE:/var/log/kadmin.log
 	admin_server = DEVICE=/dev/tty04
 .in -1i
 .fi
diff --git a/src/kadmin/cli/kadmin.M b/src/kadmin/cli/kadmin.M
index b05007a..0331bbc 100644
--- a/src/kadmin/cli/kadmin.M
+++ b/src/kadmin/cli/kadmin.M
@@ -290,7 +290,7 @@ explicitly set the key version number.
 .TP
 \fB\-policy\fP \fIpolicy\fP
 policy used by this principal.  If no policy is supplied, then if the
-policy "default" exists and the -clearpolicy is not also specified,
+policy "default" exists and the \-clearpolicy is not also specified,
 then the policy "default" is used; otherwise, the principal 
 will have no policy, and a warning message will be printed.
 .TP
@@ -633,7 +633,7 @@ Key: vno 1, DES cbc mode with CRC-32, no salt
 Key: vno 1, DES cbc mode with CRC-32, Version 4
 Attributes:
 Policy: [none]
-kadmin: getprinc -terse systest
+kadmin: getprinc \-terse systest
 systest@BLEEP.COM	3	86400	604800	1
 785926535	753241234	785900000
 tlyu/admin@BLEEP.COM	786100034	0	0
@@ -806,7 +806,7 @@ Minimum password length: 6
 Minimum number of password character classes: 2
 Number of old keys kept: 5
 Reference count: 17
-kadmin: get_policy -terse admin
+kadmin: get_policy \-terse admin
 admin	15552000	0	6	2	5	17
 kadmin:
 .TP
@@ -897,7 +897,7 @@ command.
 .RS
 .TP
 EXAMPLE:
-kadmin: ktadd -k /tmp/foo-new-keytab host/foo.mit.edu
+kadmin: ktadd \-k /tmp/foo-new-keytab host/foo.mit.edu
 Entry for principal host/foo.mit.edu@ATHENA.MIT.EDU with
 	kvno 3, encryption type DES-CBC-CRC added to keytab
 	WRFILE:/tmp/foo-new-keytab
@@ -924,7 +924,7 @@ option is specified, less verbose status information is displayed.
 .RS
 .TP
 EXAMPLE:
-kadmin: ktremove -k /usr/local/var/krb5kdc/kadmind.keytab kadmin/admin
+kadmin: ktremove \-k /usr/local/var/krb5kdc/kadmind.keytab kadmin/admin
 Entry for principal kadmin/admin with kvno 3 removed
 	from keytab WRFILE:/usr/local/var/krb5kdc/kadmind.keytab.
 kadmin:
diff --git a/src/kadmin/server/kadmind.M b/src/kadmin/server/kadmind.M
index d54b7c2..4f87788 100644
--- a/src/kadmin/server/kadmind.M
+++ b/src/kadmin/server/kadmind.M
@@ -111,7 +111,7 @@ and for which the appropriate principals are in its keytab.
 specifies that the master database password should be fetched from the
 keyboard rather than from a file on disk.  Note that the server gets the
 password prior to putting itself in the background; in combination with
-the -nofork option, you must place it in the background by hand.
+the \-nofork option, you must place it in the background by hand.
 .TP
 .B \-nofork
 specifies that the server does not put itself in the background and does
diff --git a/src/krb5-config.M b/src/krb5-config.M
index 56661ae..2fdffaa 100644
--- a/src/krb5-config.M
+++ b/src/krb5-config.M
@@ -26,37 +26,38 @@
 krb5-config \- tool for linking against MIT Kerberos libraries
 .SH SYNOPSIS
 .B krb5-config
-[ \fB--help\fP | \fB--all\fP | \fB--version\fP | \fB--vendor\fP | \fB--prefix\fP | 
-\fB--exec-prefix\fP | \fB--cflags\fP | \fB--libs\fP libraries ]
+[ \fB\-\-help\fP | \fB\-\-all\fP | \fB\-\-version\fP | \fB\-\-vendor\fP
+| \fB\-\-prefix\fP | \fB\-\-exec\-prefix\fP | \fB\-\-cflags\fP
+| \fB\-\-libs\fP libraries ]
 .br
 .SH DESCRIPTION
-.I krb5-config
+.I krb5\-config
 tells the application programmer what special flags to use to compile
 and link programs against the installed Kerberos libraries.
 .SH OPTIONS
 .TP
-\fB\--help\fP
+\fB\-\-help\fP
 print usage message.  This is the default.
 .TP
-\fB\--all\fP
+\fB\-\-all\fP
 prints version, vendor, prefix and exec-prefix.
 .TP
-\fB\--version\fP
+\fB\-\-version\fP
 prints the version of the installed Kerberos implementation.
 .TP
-\fB\--vendor\fP
+\fB\-\-vendor\fP
 prints the vendor of the installed Kerberos implementation.
 .TP
-\fB\--prefix\fP 
+\fB\-\-prefix\fP 
 prints the prefix with which Kerberos was built.
 .TP
-\fB\--exec-prefix\fP 
+\fB\-\-exec\-prefix\fP 
 prints the exec-prefix with which Kerberos was built.
 .TP
-\fB\--cflags\fP 
+\fB\-\-cflags\fP 
 prints the compiler flags with which Kerberos was built.
 .TP
-\fB\--libs\fP \fIlibraries\fP 
+\fB\-\-libs\fP \fIlibraries\fP 
 list compiler options required to link with \fIlibraries\fP.  Possible 
 values for \fIlibraries\fP are:
 .sp
diff --git a/src/plugins/kdb/ldap/ldap_util/kdb5_ldap_util.M b/src/plugins/kdb/ldap/ldap_util/kdb5_ldap_util.M
index 484c4ce..979c617 100644
--- a/src/plugins/kdb/ldap/ldap_util/kdb5_ldap_util.M
+++ b/src/plugins/kdb/ldap/ldap_util/kdb5_ldap_util.M
@@ -733,206 +733,5 @@ userpolicy
 .fi
 .RE
 
-.TP
-.B Commands Specific to eDirectory
-.TP
-\fBsetsrvpw\fP [\fB\-randpw\fP|\fB\-fileonly\fP] [\fB\-f\fP\ \fIfilename\fP] \fIservice_dn\fP
-Allows an administrator to set password for service objects such as KDC and Administration server in
-eDirectory and store them in a file. The 
-.I \-fileonly 
-option stores the password in a file and not in the eDirectory object. Options:
-.RS
-.TP
-\fB\-randpw \fP
-Generates and sets a random password. This options can be specified to store the password both in eDirectory and a file. The 
-.I \-fileonly 
-option can not be used if 
-.I \-randpw
-option is already specified.
-.TP
-\fB\-fileonly\fP
-Stores the password only in a file and not in eDirectory. The 
-.I \-randpw
-option can not be used when 
-.I \-fileonly
-options is specified.
-.TP
-\fB\-f\fP\ \fIfilename\fP
-Specifies complete path of the service password file. By default, /usr/local/var/service_passwd is used.
-.TP
-\fIservice_dn\fP
-Specifies Distinguished name (DN) of the service object whose password is to be set.
-.TP
-EXAMPLE:
-\fBkdb5_ldap_util setsrvpw \-D cn=admin,o=org setsrvpw \-fileonly \-f /home/andrew/conf_keyfile
-cn=service-kdc,o=org\fP
-.nf
-Password for "cn=admin,o=org":
-Password for "cn=service-kdc,o=org":
-Re-enter password for "cn=service-kdc,o=org":
-.fi
-.RE
-.TP
-\fBcreate_service\fP {\fB\-kdc|\-admin\fP} [\fB\-servicehost\fP\ \fIservice_host_list\fP] [\fB\-realm\fP\ \fIrealm_list\fP] [\fB\-randpw|\-fileonly\fP] [\fB\-f\fP\ \fIfilename\fP] \fIservice_dn\fP
-Creates a service in directory and assigns appropriate rights. Options:
-.RS
-.TP
-\fB\-kdc\fP 
-Specifies the service is a KDC service
-.TP
-\fB\-admin\fP
-Specifies the service is a Administration service
-.TP
-\fB\-servicehost\fP\ \fIservice_host_list\fP
-Specifies the list of entries separated by a colon (:). Each entry consists of the hostname or IP
-address of the server hosting the service, transport protocol, and the port number of
-the service separated by a pound sign (#). 
-For example,
-server1#tcp#88:server2#udp#89.
-.TP
-\fB\-realm\fP\ \fIrealm_list\fP
-Specifies the list of realms that are to be associated with this service. The list contains the name of the realms
-separated by a colon (:).
-.TP
-\fB\-randpw \fP
-Generates and sets a random password. This option is used to set the random password for the service object in directory and also to store it in the file. The 
-.I \-fileonly 
-option can not be used if 
-.I \-randpw
-option is specified.
-.TP
-\fB\-fileonly\fP
-Stores the password only in a file and not in eDirectory. The 
-.I \-randpw
-option can not be used when 
-.I \-fileonly
-option is specified.
-.TP
-\fB\-f\fP\ \fIfilename\fP
-Specifies the complete path of the file where the service object password is stashed.
-.TP
-\fIservice_dn\fP
-Specifies Distinguished name (DN) of the Kerberos service to be created.
-.TP
-EXAMPLE:
-\fBkdb5_ldap_util \-D cn=admin,o=org create_service \-kdc \-randpw \-f /home/andrew/conf_keyfile cn=service-kdc,o=org\fP
-.nf
-Password for "cn=admin,o=org":
-File does not exist. Creating the file /home/andrew/conf_keyfile...
-.fi
-.RE
-.TP
-\fBmodify_service\fP [\fB\-servicehost\fP\ \fIservice_host_list\fP | [\fB\-clearservicehost\fP\ \fIservice_host_list\fP] [\fB\-addservicehost\fP\ \fIservice_host_list\fP]] [\fB\-realm\fP\ \fIrealm_list\fP | [\fB\-clearrealm\fP\ \fIrealm_list\fP] [\fB\-addrealm\fP\ \fIrealm_list\fP]] \fIservice_dn\fP
-Modifies the attributes of a service and assigns appropriate rights. Options:
-.RS
-.TP
-\fB\-servicehost\fP\ \fIservice_host_list\fP
-Specifies the list of entries separated by a colon (:). Each entry consists of a host name
-or IP Address of the Server hosting the service, transport protocol, and port
-number of the service separated by a pound sign (#). 
-For example,
-server1#tcp#88:server2#udp#89
-.TP
-\fB\-clearservicehost\fP\ \fIservice_host_list\fP
-Specifies the list of servicehost entries to be removed from the existing list separated by colon (:). Each entry consists of a host name or IP Address of the server
-hosting the service, transport protocol, and port number of the service separated
-by a pound sign (#).
-.TP
-\fB\-addservicehost\fP\ \fIservice_host_list\fP
-Specifies the list of servicehost entries to be added to the existing list separated by colon (:). Each entry consists of a host name or IP Address of the
-server hosting the service, transport protocol, and port number of the service
-separated by a pound sign (#).
-.TP
-\fB\-realm\fP\ \fIrealm_list\fP
-Specifies the list of realms that are to be associated with this service. The list contains the name of
-the realms separated by a colon (:). This list replaces the existing list.
-.TP
-\fB\-clearrealm\fP\ \fIrealm_list\fP
-Specifies the list of realms to be removed from the existing list. The list contains the name of
-the realms separated by a colon (:).
-.TP
-\fB\-addrealm\fP\ \fIrealm_list\fP
-Specifies the list of realms to be added to the existing list. The list contains the name of the
-realms separated by a colon (:).
-.TP
-\fIservice_dn\fP
-Specifies Distinguished name (DN) of the Kerberos service to be modified.
-.TP
-EXAMPLE:
-\fBkdb5_ldap_util \-D cn=admin,o=org modify_service \-realm ATHENA.MIT.EDU
-cn=service-kdc,o=org\fP
-.nf
-Password for "cn=admin,o=org":
-Changing rights for the service object. Please wait ... done
-.fi
-.RE
-.TP
-\fBview_service\fP \fIservice_dn\fP
-Displays the attributes of a service.  Options:
-.RS
-.TP
-\fIservice_dn\fP
-Specifies Distinguished name (DN) of the Kerberos service to be viewed.
-.TP
-EXAMPLE:
-\fBkdb5_ldap_util \-D cn=admin,o=org view_service cn=service-kdc,o=org\fP
-.nf
-Password for "cn=admin,o=org":
-        Service dn: cn=service-kdc,o=org
-      Service type: kdc
- Service host list:
-     Realm DN list: cn=ATHENA.MIT.EDU,cn=Kerberos,cn=Security
-.fi
-.RE
-.TP
-\fBdestroy_service\fP [\fB\-force\fP] [\fB\-f\fP\ \fIstashfilename\fP] \fIservice_dn\fP
-Destroys an existing service. Options:
-.RS
-.TP
-\fB\-force\fP
-If specified, will not prompt for user's confirmation, instead will force destruction of the service.
-.TP
-\fB\-f\fP\ \fIstashfilename\fP
-Specifies the complete path of the service password file from where the entry corresponding to the 
-.I service_dn
-needs to be removed.
-.TP
-\fIservice_dn\fP
-Specifies Distinguished name (DN) of the Kerberos service to be destroyed.
-.TP
-EXAMPLE:
-\fBkdb5_ldap_util \-D cn=admin,o=org destroy_service cn=service-kdc,o=org\fP
-.nf
-Password for "cn=admin,o=org":
-This will delete the service object 'cn=service-kdc,o=org', are you sure?
-(type 'yes' to confirm)? yes
-** service object 'cn=service-kdc,o=org' deleted.
-.fi
-.RE
-.TP
-\fBlist_service\fP [\fB\-basedn\fP\ \fIbase_dn\fP]
-Lists the name of services under a given base in directory. Options:
-.RS
-.TP
-\fB\-basedn\fP\ \fIbase_dn\fP
-Specifies the base DN for searching the service objects, limiting the search to a particular subtree. If this option
-is not provided, LDAP Server specific search base will be used. 
-For eg, in the case of OpenLDAP, value of 
-.B defaultsearchbase
-from 
-.I slapd.conf
-file will be used, where as in the case of eDirectory, the default value 
-for the base DN is 
-.B Root.
-.TP
-EXAMPLE:
-\fBkdb5_ldap_util \-D cn=admin,o=org list_service\fP
-.nf
-Password for "cn=admin,o=org":
-cn=service-kdc,o=org
-cn=service-adm,o=org
-cn=service-pwd,o=org
-.fi
-.RE
 .SH SEE ALSO
 kadmin(8)
diff --git a/src/slave/kprop.M b/src/slave/kprop.M
index cb4ae56..3a9ea85 100644
--- a/src/slave/kprop.M
+++ b/src/slave/kprop.M
@@ -39,7 +39,7 @@ Kerberos server to a slave Kerberos server, which is specified by
 This is done by transmitting the dumped database file to the slave
 server over an encrypted, secure channel.  The dump file must be created
 by kdb5_util, and is normally KPROP_DEFAULT_FILE
-(/usr/local/var/krb5kdc/slave_datatrans).
+(/var/lib/krb5kdc/slave_datatrans).
 .SH OPTIONS
 .TP
 \fB\-r\fP \fIrealm\fP
@@ -51,7 +51,7 @@ is used.
 \fB\-f\fP \fIfile\fP
 specifies the filename where the dumped principal database file is to be
 found; by default the dumped database file is KPROP_DEFAULT_FILE
-(normally /usr/local/var/krb5kdc/slave_datatrans).
+(normally /var/lib/krb5kdc/slave_datatrans).
 .TP
 \fB\-P\fP \fIport\fP
 specifies the port to use to contact the
diff --git a/src/slave/kpropd.M b/src/slave/kpropd.M
index f3283c4..4284048 100644
--- a/src/slave/kpropd.M
+++ b/src/slave/kpropd.M
@@ -74,7 +74,7 @@ Normally, kpropd is invoked out of
 This is done by adding a line to the inetd.conf file which looks like
 this:
 
-kprop	stream	tcp	nowait	root	/usr/local/sbin/kpropd	kpropd
+krb5_prop	stream	tcp	nowait	root	/usr/sbin/kpropd	kpropd
 
 However, kpropd can also run as a standalone daemon, if the
 .B \-S
@@ -111,13 +111,13 @@ is used.
 \fB\-f\fP \fIfile\fP
 specifies the filename where the dumped principal database file is to be
 stored; by default the dumped database file is KPROPD_DEFAULT_FILE
-(normally /usr/local/var/krb5kdc/from_master).
+(normally /var/lib/krb5kdc/from_master).
 .TP
 .B \-p
 allows the user to specify the pathname to the
 .IR kdb5_util (8)
 program; by default the pathname used is KPROPD_DEFAULT_KDB5_UTIL
-(normally /usr/local/sbin/kdb5_util).
+(normally /usr/sbin/kdb5_util).
 .TP
 .B \-S
 turn on standalone mode.  Normally, kpropd is invoked out of
@@ -148,14 +148,14 @@ mode.
 allows the user to specify the path to the
 kpropd.acl
 file; by default the path used is KPROPD_ACL_FILE
-(normally /usr/local/var/krb5kdc/kpropd.acl).
+(normally /var/lib/krb5kdc/kpropd.acl).
 .SH FILES
 .TP "\w'kpropd.acl\ \ 'u"
 kpropd.acl
 Access file for
 .BR kpropd ;
 the default location is KPROPD_ACL_FILE (normally
-/usr/local/var/krb5kdc/kpropd.acl).
+/var/lib/krb5kdc/kpropd.acl).
 Each entry is a line containing the principal of a host from which the
 local machine will allow Kerberos database propagation via kprop.
 .SH SEE ALSO
diff --git a/src/slave/kproplog.M b/src/slave/kproplog.M
index b7081a9..778d001 100644
--- a/src/slave/kproplog.M
+++ b/src/slave/kproplog.M
@@ -24,7 +24,7 @@
 .\"
 .\" Copyright (c) 2003, Sun Microsystems, Inc.  All Rights Reserved
 .\"
-.TH KPROPLOG 1
+.TH KPROPLOG 8
 .SH NAME
 kproplog \- display the contents of the Kerberos principal update log
 .SH SYNOPSIS
